bundles/org.openhab.binding.nanoleaf/README.md

```diff
@@ -4,38 +4,76 @@ This binding integrates the [Nanoleaf Light Panels](https://nanoleaf.me/en/consu
 
+
 The binding supports pairing of the device with your openHAB instance as follows:
 
 1. Make sure that the authentication token field in your Nanoleaf controller thing configuration is left empty.
-2. Hold down the on-off button of the controller for 5-7 seconds until the LED starts flashing in a pattern, which turns the device in pairing mode, and openHAB will try to request an authentication token for it.
+2. Hold down the on-off button of the controller for 5-7 seconds until the LED starts flashing/cycling in a pattern, which turns the device in pairing mode, and openHAB will try to request an authentication token for it.
 
 Once your openHAB instance successfully requested and stored the authentication token in the controller's thing configuration, the controller status changes to ONLINE, and you can start linking the channels to your items.
 
-For a successfully paired and online controller, the individual panels connected to it can be discovered by starting another scan for the Nanoleaf binding (e.g. from the Inbox in Paper UI). 
+Tip: if you press (2) just before adding the item from the inbox it usually catches the auth token right away and if you are lucky it already automatically starts discovering the panels in one turn (see below).
+
+**Adding the invidual light panels as a thing**
+
+After you have added the controller as a thing and it has been successfully paired as described as above, the individual panels connected to it can be discovered by **starting another scan** for the Nanoleaf binding (e.g. from the Inbox in Paper UI). 
 All connected panels will be added as separate things to the inbox.
 
+Troubleshooting: In seldom cases (in particular together with updating the binding) things or items do not work as expected, are offline or may not be detected. In this case
+
+- remove the panels (maybe twice by force removing it)
+- remove the controller (maybe twice by force removing it)
+- stop and then start openhab
+- Rediscover like described above
+
+**Knowing which panel has which id**
+
+Unfortunately it is not easy to find out which panel gets which id while this is pretty important if you have lots of them and you want to assign rules to it. 
+Don't worry: the binding comes with some helpful support in the background (this works only well for the canvas device!)
+
+- fire up your browser and open the openhab server on port 9001 which shows the logs.
+- look out for something like "Panel layout and ids" in the logs. Below that you will see a panel layout similar to
+```                                     
+                                    41451                                     
+                               
+                        58086        8134                   39755             
+
+55836       56093       48111       38724       17870        5164       64279 
+
+            31413                    9162       13276     
+```                 
+
+
 ## Thing Configuration
```

Comment:
The new config property "deviceType" should be added to the table

Reply:
Hi Martin, I am not really sure what you mean by that. What table do you mean?